scripts/install.sh

```diff
@@ -93,6 +93,21 @@ configure_systemd() {
         status "Adding ollama user to video group..."
         $SUDO usermod -a -G video ollama
     fi
+    status "Creating ollama systemd service directory if not exists..."
+    $SUDO mkdir -p /etc/systemd/system/ollama.service.d
+
+    status "Creating environment.conf file if not exists..."
+    ENV_CONF_FILE="/etc/systemd/system/ollama.service.d/environment.conf"
+    if [ ! -f "$ENV_CONF_FILE" ]; then
+        cat <<EOF | $SUDO tee "$ENV_CONF_FILE" >/dev/null
+[Service]
+# Conditionally include EnvironmentFile directive
+EnvironmentFile=/etc/default/ollama
```

Comment:
`EnvironmentFile` should be in the main service file. It minimizes drift making it easier to support

Reply:
Someone did this instead:
ENV_CONF_FILE="/etc/systemd/system/ollama.service.d/environment.conf"
And THAT file is pointing to the /etc/default/ollama file.  So someone went around the direct approach.  Should I just kill this pull request?